### ISSUE TITLE
precedence of \over and \middle

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#49]
Example
```tex
\left\{a\over2\middle|b\right\}
```
In [LaTeX](https://texlive.net/run?%5Cdocumentclass%7Barticle%7D%0A%5Cusepackage%7Bamsmath%7D%0A%5Cbegin%7Bdocument%7D%0A%24%5Cleft%5C%7Ba%5Cover2%5Cmiddle%7Cb%5Cright%5C%7D%24%0A%5Cend%7Bdocument%7D)
<p align="center"><img src="https://github.com/ronkok/Temml/assets/53823634/8c7716f7-a481-4e19-86cf-585f7f159348"></p>
In <a href="https://temml.org/#%5Cleft%5C%7Ba%5Cover2%5Cmiddle%7Cb%5Cright%5C%7D">Temml</a>
<img src="https://github.com/ronkok/Temml/assets/53823634/62021bff-eb66-4139-ba81-fcda0fa678ac">

Similarly
```tex
\left\{a\over2\middle/b\right\}
```
In [LaTeX](https://texlive.net/run?%5Cdocumentclass%7Barticle%7D%0A%5Cusepackage%7Bamsmath%7D%0A%5Cbegin%7Bdocument%7D%0A%24%5Cleft%5C%7Ba%5Cover2%5Cmiddle/b%5Cright%5C%7D%24%0A%5Cend%7Bdocument%7D)
<p align="center"><img src="https://github.com/ronkok/Temml/assets/53823634/357850ee-eb7b-42c0-9635-d76e5ea8f397"></p>

In <a href="https://temml.org/#%5Cleft%5C%7Ba%5Cover2%5Cmiddle/b%5Cright%5C%7D">Temml</a>
![image](https://github.com/ronkok/Temml/assets/53823634/e16f3842-f078-4879-be21-91a439a9c0db)

**Comment by ronkok** [comment]
Yikes! I was unaware that `\middle` has that kind of effect.

This will require some work in a tricky part of the parser. Some days may go by before I can fix it.

**Comment by ronkok** [comment]
I think it is interesting to note that this particular parsing bug has been in KaTeX since `\middle` was introduced 8 years ago. In all that time, no one has reported this bug.

@hbghlyj You're doing some very thorough work.

**Comment by ronkok** [comment]
Fixed by release v0.10.22.
